debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -322,11 +320,13 @@ public static SchemaNameAdjustmentMode parse(String value) {
             .withGroup(Field.createGroupEntry(Field.Group.CONNECTION, 0))
             .withWidth(Width.MEDIUM)
             .withImportance(Importance.HIGH)
-            .withDefault(LOGIC_NAME_PLACEHOLDER)
             .withValidation(CommonConnectorConfig::validateTopicName)
             .required()
-            .withDescription("The name of the prefix to be used for all topics, the placeholder " + LOGIC_NAME_PLACEHOLDER +
-                    " can be used for referring to the connector's logical name as default value.");
+            .withDescription("Topic prefix that provides a namespace for the particular MySQL database " +
```

Comment:
This is common config, so we shouldn't mention MySQL here.

Reply:
@vjuranek 
Revert this config description.